<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.8.10 on linux)" generated="20240119 11:06:37.077" rpa="false" schemaversion="4">
<suite id="s1" name="Flipkart" source="/home/ridham/PycharmProjects/MobileAutomation/Exercise1/Tests/Flipkart.robot">
<test id="s1-t1" name="Open Test Application" line="23">
<kw name="Open Application" library="AppiumLibrary">
<arg>${APPIUM_SERVER}</arg>
<arg>automationName=${AUTOMATION_NAME}</arg>
<arg>platformName=${PLATFORM_NAME}</arg>
<arg>platformVersion=${PLATFORM_VERSION}</arg>
<arg>deviceName=${DEVICE_NAME}</arg>
<arg>appPackage=${PACKAGE_NAME}</arg>
<arg>appActivity=${ACTIVITY_NAME}</arg>
<arg>noReset=true</arg>
<arg>allowInsecure=true</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<msg timestamp="20240119 11:06:37.586" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f93801dd670&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session</msg>
<msg timestamp="20240119 11:06:37.586" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f93801dda00&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session</msg>
<msg timestamp="20240119 11:06:37.587" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f938017b2e0&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session</msg>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20240119 11:06:37.588" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20240119 11:06:37.588" endtime="20240119 11:06:37.651"/>
</kw>
<msg timestamp="20240119 11:06:37.664" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20240119 11:06:37.665" level="FAIL">MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f938017b430&gt;: Failed to establish a new connection: [Errno 111] Connection refused'))</msg>
<status status="FAIL" starttime="20240119 11:06:37.582" endtime="20240119 11:06:37.669"/>
</kw>
<status status="FAIL" starttime="20240119 11:06:37.566" endtime="20240119 11:06:37.670">MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f938017b430&gt;: Failed to establish a new connection: [Errno 111] Connection refused'))</status>
</test>
<test id="s1-t2" name="Select Item" line="34">
<kw name="Sleep" library="BuiltIn">
<arg>5s</arg>
<doc>Pauses the test executed for the given time.</doc>
<msg timestamp="20240119 11:06:42.675" level="INFO">Slept 5 seconds</msg>
<status status="PASS" starttime="20240119 11:06:37.674" endtime="20240119 11:06:42.676"/>
</kw>
<kw name="Swipe" library="AppiumLibrary">
<arg>300</arg>
<arg>2200</arg>
<arg>300</arg>
<arg>150</arg>
<doc>Swipe from one point to another point, for an optional duration.</doc>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20240119 11:06:42.676" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20240119 11:06:42.676" endtime="20240119 11:06:42.677"/>
</kw>
<msg timestamp="20240119 11:06:42.679" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20240119 11:06:42.679" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20240119 11:06:42.676" endtime="20240119 11:06:42.679"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>2s</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.680" endtime="20240119 11:06:42.680"/>
</kw>
<kw name="Swipe" library="AppiumLibrary">
<arg>300</arg>
<arg>2200</arg>
<arg>300</arg>
<arg>150</arg>
<doc>Swipe from one point to another point, for an optional duration.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.680" endtime="20240119 11:06:42.680"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>2s</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.681" endtime="20240119 11:06:42.681"/>
</kw>
<kw name="Swipe" library="AppiumLibrary">
<arg>300</arg>
<arg>2200</arg>
<arg>300</arg>
<arg>150</arg>
<doc>Swipe from one point to another point, for an optional duration.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.681" endtime="20240119 11:06:42.681"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>2s</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.681" endtime="20240119 11:06:42.682"/>
</kw>
<kw name="Wait Until Element Is Visible" library="AppiumLibrary">
<arg>${ITEM}</arg>
<arg>${TIMEOUT}</arg>
<doc>Waits until element specified with `locator` is visible.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.682" endtime="20240119 11:06:42.682"/>
</kw>
<kw name="Click Element" library="AppiumLibrary">
<arg>${ITEM}</arg>
<doc>Click element identified by `locator`.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.682" endtime="20240119 11:06:42.682"/>
</kw>
<kw name="Wait Until Element Is Visible" library="AppiumLibrary">
<arg>${BUY}</arg>
<arg>${TIMEOUT}</arg>
<doc>Waits until element specified with `locator` is visible.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.682" endtime="20240119 11:06:42.682"/>
</kw>
<kw name="Click Element" library="AppiumLibrary">
<arg>${BUY}</arg>
<doc>Click element identified by `locator`.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.682" endtime="20240119 11:06:42.682"/>
</kw>
<kw name="Click Element" library="AppiumLibrary">
<arg>${CONTIUNE}</arg>
<doc>Click element identified by `locator`.</doc>
<status status="NOT RUN" starttime="20240119 11:06:42.682" endtime="20240119 11:06:42.682"/>
</kw>
<status status="FAIL" starttime="20240119 11:06:37.672" endtime="20240119 11:06:42.682">No application is open</status>
</test>
<status status="FAIL" starttime="20240119 11:06:37.078" endtime="20240119 11:06:42.683"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="2" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="2" skip="0" id="s1" name="Flipkart">Flipkart</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20240119 11:06:37.586" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f93801dd670&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session</msg>
<msg timestamp="20240119 11:06:37.586" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f93801dda00&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session</msg>
<msg timestamp="20240119 11:06:37.587" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7f938017b2e0&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session</msg>
<msg timestamp="20240119 11:06:37.664" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20240119 11:06:42.679" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
